docs(snackbar, toast): update docs according to the latest changes (#2870)
IgniteUI/igniteui-docfx: en/components/snackbar.md

_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Snackbar component, Angular Snackbar control
---
# Snackbar

> <p class="highlight">The Ignite UI for Angular Snackbar component provides feedback about an operation with a single-line message, which can include a link to an action such as Undo. The Snackbar message appears above all other screen elements, located at the bottom of a mobile device screen or at the lower left of larger device screens.</p>

Reviewer: ```suggestion
<p class="highlight">The Ignite UI for Angular Snackbar component provides feedback about an operation with a single-line message, which can include an action. The Snackbar message appears above all other elements and is positioned at the bottom center of the screen.</p>
```

// sample.component.ts

public close(element) {
    element.close();
}
```

> If the sample is configured properly, the first snackbar appears when the button is clicked, showing both the *message* and *action button*. The auto-hide feature is disabled and the snackbar disappears on 'CLOSE' button click. Another snackbar passes a different message through the [`open()`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html#open) method and hide it when the *display time* ends. The third component passes a message as a param to the [`open()`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html#open) method and adds an icon using the content projection.

Reviewer: ```suggestion
If the sample is configured properly, the first snackbar appears when the button is clicked, showing both the *message* and *action button*. The auto-hide feature is disabled and the snackbar disappears on 'CLOSE' button click. Another snackbar passes a different message through the [`open()`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html#open) method and hides it when the *display time* expires. The third component passes a message as a param to the [`open()`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html#open) method and adds an icon using content projection.
```
